src/main/scala/com/microsoft/hyperspace/index/rules/RuleUtils.scala

```diff
@@ -71,7 +71,10 @@ object RuleUtils {
       }
     }
 
-    def isHybridScanCandidate(entry: IndexLogEntry, inputSourceFiles: Seq[FileInfo]): Boolean = {
+    def isHybridScanCandidate(
+        entry: IndexLogEntry,
+        inputSourceFiles: Seq[FileInfo],
+        totalInputBytes: Long): Boolean = {
```

Comment:
To imply that `inputSourceFiles` and `totalInputBytes` are related, we can either make a pair, or have `inputSourceFiles` and `inputSourceFilesSizeInBytes`.